core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -214,14 +214,14 @@ object TestUtils extends Logging {
     enableToken: Boolean = false,
     numPartitions: Int = 1,
     defaultReplicationFactor: Short = 1,
-    startingIdNumber: Int = 0
-  ): Seq[Properties] = {
+    startingIdNumber: Int = 0,
+    enableFetchFromFollower: Boolean = false): Seq[Properties] = {
```

Comment:
This method has far too many parameters, it doesn't make sense to add parameters for every single possible configuration. Instead, you can add whatever property you want to the returned `Properties` instance.